docs/changes.rst

```diff
@@ -10,6 +10,7 @@ Pending
 * Wrap ``SHOW_TOOLBAR_CALLBACK`` function with ``sync_to_async``
   or ``async_to_sync`` to allow sync/async compatibility.
 * Make ``require_toolbar`` decorator compatible to async views.
+* Added link to contributing documentation in CONTRIBUTING.md to improve contributor onboarding.
```

Comment:
```suggestion
* Added link to contributing documentation in `CONTRIBUTING.md`.
```

The spell checker doesn't seem to like "onboarding", I wouldn't know why. I think if we shorten it we don't lose clarity and the spellchecker should be happy, or so I hope! 

This should also keep the line within the required maximum length. You could also wrap the changelog entry if you prefer the longer version, but then you'd probably also have to add the offending word to `docs/spelling_wordlist.txt`.

Reply:
Thank you 😄